org.eclipse.winery.repository/src/main/java/org/eclipse/winery/repository/backend/filebased/GitBasedRepository.java

```diff
@@ -39,11 +39,12 @@
 public class GitBasedRepository extends FilebasedRepository {
 
 	/**
-	 * Used for synchronizing the method {@link GitBasedRepository#addCommit()}
+	 * Used for synchronizing the method {@link GitBasedRepository#addCommit(RepositoryFileReference)}
 	 */
 	private static final Object COMMIT_LOCK = new Object();
 	private static final Logger LOGGER = LoggerFactory.getLogger(GitBasedRepository.class);
 
+	private final Repository gitRepo;
```

Comment:
Why is this variable a class variable? Isn't it used in a single method only? Will there be a future use of that class variable?